src/kaleidoscope/driver/led/Base.h

```diff
@@ -25,8 +25,14 @@ namespace kaleidoscope {
 namespace driver {
 namespace led {
 
+constexpr uint8_t no_led = 254;
```

Comment:
At the moment, the Imago uses 254, while the Raise uses 255. We should figure out if this needs to be device specific, or can be generic, and if the latter, what value should it have? It's (currently) not checked by anything as far as I see. We should probably fix that too. (Adding checks is a different PR, just mentioning here)

I'd also suggest a different name, to make it clear what this is used for: `no_led_at_key`. Though, that might be a bit on the too long side of things.

Reply:
Is it "safe" to declare it as -1 globally, as a shorthand for "last value in the unsigned datatype" or will the compiler be smart and stop us from doing that?

Reply:
`uint8_t(-1)` should work. That's a common idiom for what you intend.

Reply:
@obra, what's your opinion about the name of the no_led constant?

Reply:
Pushed an update that implements the -1 solution.